underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java

```diff
@@ -552,17 +535,63 @@ private boolean isDirectory(String path) throws IOException {
   }
 
   /**
-   * Lists the files or folders which match the given prefix.
+   * Lists the files or folders in the given path, not including the path itself.
+   *
+   * @param path the folder path whose children are listed
```

Comment:
No. The UnderFileSystem specifies that the path must be a directory. If path does not exist as directory, return null.